constants/colors.ts

```diff
@@ -7,6 +7,7 @@ export const red550 = "hsla(352, 23%, 25%, 1)";
 // Whites
 export const white = "hsla(0, 0%, 100%, 1)"; // #FFFFFF
 export const whiteOpacity10 = "hsla(0, 0%, 100%, 0.1)";
```

Comment:
I've also been getting frustrated with adding lots of variables for different opacity levels in the voter dapp code, so I created this helper function

`export function addOpacityToHsl(hsl: string, opacity: number) {
  const betweenParens = hsl.match(/\(([^)]+)\)/)?.[1];
  const [h, s, l] = betweenParens?.split(",") ?? [];
  return `hsla(${h}, ${s}, ${l}, ${opacity})`;
}
`

It lets you take one of these color vars and add whatever opacity you want. I reckon I'm going to start using this instead for most of the places that need colors with opacity, although I will keep some of the ones that get used very often for convenience. You might want to use it as well.